### PR TITLE
Dawn/build ia updates

### PR DESCRIPTION
**Pull request by dawnkelly09**
This PR is step 1 in updating the IA for the Build | Develop | Deploy section:
- Adds needed folders and placeholder pages for new IA
- Removes all old files except content md files which I will move under separate PR

### CODE REVIEW

File: build-deploy-develop/toolkit/api-libraries/.pages [hunk 1]
```diff
@@ -0,0 +1,7 @@
+title: API Libraries
+nav:
+  - index.md
+  - 'Polkadot.js SDK (JavaScript/TypeScript)': polkadot-js-sdk.md
+  - 'PAPI (JavaScript/TypeScript)': papi.md
+  - 'Sidecar (Rest API)': sidecar.md
+  - 'PY-Substrate-Interface (Python)': py-substrate-interface.md
```
Comment (eshaben) [inline build-deploy-develop/toolkit/api-libraries/.pages:7]:
```suggestion
  - 'Polkadot.js SDK (JavaScript/TypeScript)': polkadot-js-sdk.md
  - 'PAPI (JavaScript/TypeScript)': papi.md
  - 'Sidecar (Rest API)': sidecar.md
  - 'Python Substrate Interface (Python)': py-substrate-interface.md
```

It's kinda weird it says python twice. I'd honestly prefer if we didn't use the parenthesis in the titles. But I'm just gonna go with the flow here. Can always change later if need be 🤷‍♀️

File: build-deploy-develop/toolkit/api-libraries/py-substrate-interface.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+---
+title: PY-Substrate-Interface (Python)
```
Comment (eshaben) [inline build-deploy-develop/toolkit/api-libraries/py-substrate-interface.md:2]:
```suggestion
title: Python Substrate Interface (Python)
```

File: build-deploy-develop/toolkit/blockchain/.pages [hunk 3]
```diff
@@ -0,0 +1,6 @@
+title: Blockchain
+nav:
+  - index.md
+  - 'Spawn Chains for Testing (Zombienet)': spawn-testing-chain.md
```
Comment (eshaben) [inline build-deploy-develop/toolkit/blockchain/.pages:4]:
```suggestion
  - 'Spawn Chains for Testing with Zombienet': spawn-testing-chain.md
```

File: build-deploy-develop/toolkit/blockchain/.pages [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+  - 'Fork Live Chains (Chopsticks)': fork-live-chains.md
```
Comment (eshaben) [inline build-deploy-develop/toolkit/blockchain/.pages:5]:
```suggestion
  - 'Fork Live Chains with Chopsticks': fork-live-chains.md
```

File: .pages [hunk 5]
```diff
@@ -1,6 +1,6 @@
 nav:
   - index.md
-  - develop
+  - build-deploy-develop
```
Comment (eshaben) [inline .pages:3]: this should be changed

File: build-deploy-develop/.pages [hunk 6]
```diff
@@ -0,0 +1,10 @@
+title: Build | Deploy | Develop
```
Comment (eshaben) [inline build-deploy-develop/.pages:1]: this should be changed

**Review by eshaben — COMMENTED** [review]
Nice! Thanks!

**Review by eshaben — APPROVED** [review]
Sweeet... LGTM!

File: develop/blockchains/deployment/generate-chain-specs.md [hunk 7]
```diff
@@ -0,0 +1,4 @@
+---
+title: Generate Chain Specs and Wasm
```
Comment (CrackTheCode016) [inline develop/blockchains/deployment/generate-chain-specs.md:2]: FYI, there are usually three different things when you deploy: 

- genesis head
- the wasm
- the chain spec (which is generated from the wasm)

```suggestion
title: Generate Genesis, Chain Specifications and Wasm
```

**Review by CrackTheCode016 — APPROVED** [review]
lgtm!

**Review by eshaben — APPROVED** [review]

**Pull request merged**